app/views/katello/api/v2/organizations/show.json.rabl

```diff
@@ -7,6 +7,9 @@ attributes :task_id, :label, :owner_details, :redhat_repository_url
 if ::SETTINGS[:katello][:use_cp]
   attributes :system_purposes, :system_purposes
   attributes :service_levels, :service_level
+  node :simple_content_access do |org|
+    org.golden_ticket?
```

Comment:
Note this method got renamed to ```org.simple_content_access?```

Reply:
@parthaa Yes, looks like I do have to change it since #8529 is merged now.  Thanks :)